pilot/pkg/networking/plugin/mixer/mixer.go

```diff
@@ -65,13 +68,10 @@ func (mixerplugin) OnOutboundListener(in *plugin.InputParams, mutable *plugin.Mu
 	}
 
 	attrs := attributes{
-		"source.namespace": attrNamespace(in.Node),
-	}
-
-	if has10Config(in.Node) {
```

Comment:
fwiw, this kept the check very localized. Mixer's interpretation of 1.0 may be different from Auth's interpretation of 1.0. That said, if we are going to move the version thing to common model, then pleaes follow the suggestion above, and make it return the version string, instead of doing a boolean check for version X or version Y, as version interpretations are different.

Reply:
There is nothing specific about mixer here. Checking proxy version is a core pilot requirement.

Reply:
I don't understand. Why is 1.0 subject to interpretation? Are we shipping multiple 1.0 proxies?